Fix for Jazzcat to handle scripts that have onload handlers attached

jansepar opened the pull request.

jansepar committed:
Fix for Jazzcat to handle scripts that have onload handlers attached

jansepar committed:
Improve onload handling for scripts

jansepar committed:
Small fix

noahadams commented on src/jazzcat.js:
```diff
@@ -388,8 +388,14 @@ define(["mobifyjs/utils", "mobifyjs/capture"], function(Utils, Capture) {
+                // ex. <script>Jazzcat.exec("http://code.jquery.com/jquery.js")</script>
+                if (script.hasAttribute('onload')){
+                    var onload = script.getAttribute('onload');
+                    script.innerHTML =  options.execCallback + "('" + url + "', '" + onload.replace(/'/g, '\\\'') + "');";
```
What's with the single quote escaping of the onload handler?

jansepar replied:
Since we're wrapping the value with single quotes, we need to ensure that what is in the onload string doesn't contain single quotes. We're constructing a string version of the exec call.

noahadams committed:
code style improvements

- added semi-colons
- indented visually
- add radix parameter to call to parseInt

jansepar committed:
Merge remote-tracking branch 'origin/v2.0' into v2.0-jazzcat-onload-fix

Conflicts:
	build/mobify.min.js

jansepar commented on package.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "mobifyjs",
-  "version": "2.0.0alpha7",
+  "version": "2.0.0alpha9",
```
Just a heads up, this is because for my last tag release, I forgot to bump the version, thus the increase by 2.

noahadams committed:
update examples and built files

noahadams commented:
@jansepar I did a little code cleanup on the file and added to the example, if you're cool with those you have my :+1:

jansepar committed:
Update build files

jansepar committed:
Update build files after new grunt-requirejs was installed

jansepar commented:
Your changes look good to me! Thanks @noahadams
